docs/bnb_smart_chain/validator/manage-keys.md

```diff
@@ -0,0 +1,53 @@
+# Key Management for BSC Validator
```

Comment:
The run-val.md file have breaking link

Reply:
Yes, but the fullnode runbook is not in this scope, should fix it after added the fullnode part.

Reply:
fixed